From 234c185e2909622fe6cbc36955f238b261e7dff2 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Fri, 22 Mar 2019 09:14:43 -0700
Subject: [PATCH] TRITON-1345 fix sdc-manta conditional inclusion of
 agent_prebuilt Makefile so can 'make check' on non-SmartOS

---
 Makefile | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 23ca3ca..aad8213 100644
--- a/Makefile
+++ b/Makefile
@@ -171,8 +171,10 @@ publish: release
 CLEAN_FILES += node_modules
 
 include ./deps/eng/tools/mk/Makefile.deps
-include ./deps/eng/tools/mk/Makefile.node_prebuilt.targ
-include ./deps/eng/tools/mk/Makefile.agent_prebuilt.targ
+ifeq ($(shell uname -s),SunOS)
+    include ./deps/eng/tools/mk/Makefile.node_prebuilt.targ
+    include ./deps/eng/tools/mk/Makefile.agent_prebuilt.targ
+endif
 include ./deps/eng/tools/mk/Makefile.targ
 
 MAN_SECTION	:= 1
-- 
2.21.0

